content/webapp/components/EventsByMonth/group-event-utils.test.ts

```diff
@@ -1,275 +1,169 @@
-import {
-  createLabel,
-  findMonthsThatEventSpans,
-  getMonthsInDateRange,
-  groupEventsByMonth,
-  sortByEarliestFutureDateRange,
-} from './group-event-utils';
-
-describe('createLabel', () => {
-  it('zero-pads the month value', () => {
-    expect(createLabel({ year: 2001, month: 1 })).toEqual('2001-02');
-  });
-
-  it('gets the right label', () => {
-    expect(createLabel({ year: 2001, month: 11 })).toEqual('2001-12');
-  });
-});
+import { getMonthsInDateRange, groupEventsByMonth } from './group-event-utils';
 
 describe('getMonthsInDateRange', () => {
   it('finds a single month', () => {
     const result = getMonthsInDateRange({
-      start: new Date(2001, 1, 1),
-      end: new Date(2001, 1, 5),
+      start: new Date('2001-01-01'),
+      end: new Date('2001-01-05'),
```

Comment:
It turns out these aren't the same, which was almost certainly behind some of the confusion and bugs.

```
> new Date(2001, 1, 1)
Feb 01 2001
```